packages/components/components/elvis-timepicker/src/react/elvia-timepicker.tsx

```diff
@@ -214,7 +228,7 @@ export const Timepicker: React.FC<Partial<TimepickerProps>> = ({
         isInvalid={!!error || !!mergedErrorOptions.text || !!mergedErrorOptions.isErrorState}
         hasErrorPlaceholder={!!error || !!mergedErrorOptions.hasErrorPlaceholder || !!mergedErrorOptions.text}
       >
-        {!!label && <FormFieldLabel data-testid="label">{label}</FormFieldLabel>}
+        {!!labels.label && <FormFieldLabel data-testid="label">{labels.label}</FormFieldLabel>}
         <FormFieldInputContainer ref={connectedElementRef}>
           <TimepickerInput
```

Comment:
Notice that inside `TimepickerInput` there is a placeholder. The placeholder should also be translated, as we did inside the date picker:

"tt:mm:ss" <-> "hh:mm:ss"